### DISCUSSION TITLE
실행 중 마우스 깜빡임 현상

### DISCUSSION THREAD

**Discussion by IronKoh** [metamong-friends/metamong-friends.github.io#44] [제보하기]
메타몽이 실행되고 있는 동안에
마우스 커서 깜빡임 현상이 발생합니다. (필요 시 영상으로 제공해드릴게요)

메타몽을 종료하면 깜빡임 현상이 사라집니다.

**Comment by ACT-TJ** [comment]
안녕하세요!

에이든 께서도 직접 보여주셨고, 저희도 일부 확인을 한 부분이라 ㅠㅠ 
저희도 해당 문제를 인지하고 조치를 진행하고 있으나,
특정 결함이 아닌 Application 최적화와 연관이 있는 것으로 보고있어
시간이 좀 걸리고 있습니다. 
사용 메모리 최적화는 일부 마쳐 릴리즈 하였으며,
CPU 는 Frame Rate 와 연관이 커서 직접 조정할 수 있도록 설정 쪽에 기능을 추가하였습니다.

GPU 사용과 가장 연관이 커 보이는데, 시간이 걸리더라도 조금씩이라도 개선이 되는 대로 릴리즈를 하고 
공지 드리도록 하겠습니다.

향후 조금씩 나아지는 모습이 보이거나 개선되는 부분도 함께 피드백 주시면 더욱 감사할 것 같습니다.^^

**Comment by shlee303** [comment]
@IronKoh 혹시 해당 깜빡임 이슈 지속 발생되는지 확인 부탁드려도 될까요?
